src/components/molecules/Results.vue

```diff
@@ -36,17 +38,54 @@ export default defineComponent({
     :headers="headers"
     :items="searchResults"
     class="resultsTable"
+    calculate-widths
   >
     <!-- eslint-disable-next-line -->
      <template #item.id-name="{item}">
       <div
         class="text-truncate"
         style="max-width: 10vw;"
       >
-        <a :href="`${baseLink}${item.spatial_id}`">
-          {{ item.spatial_id }} - {{ item.subentry_name }}
-        </a>
+        <v-tooltip bottom>
+          <template v-slot:activator="{ on, attrs }">
+            <a
+              :href="`${baseLink}${item.spatial_id}`"
+              v-bind="attrs"
+              v-on="on"
+            >
+              {{ item.spatial_id }}{{ item.subentry_name ? ` - ${item.subentry_name}` : '' }}
+            </a>
+          </template>
+          <span>
+            View {{ item.spatial_id }}{{ item.subentry_name ? ` - ${item.subentry_name}` : '' }}
+          </span>
+        </v-tooltip>
       </div>
     </template>
+    <!-- eslint-disable-next-line -->
+    <template #item.subentry_type="{item}">
+      <v-tooltip bottom>
+        <template v-slot:activator="{ on, attrs }">
+          <img
+            v-bind="attrs"
+            :src="`icons/${item.subentry_type.toLowerCase()}.png`"
+            :alt="item.subentry_type"
+            class="data-type-icon"
+            v-on="on"
+          >
+        </template>
+        <span>
+          {{ item.subentry_type }}
```

Comment:
I would actually have the icon replace the `subentry_type`

Reply:
The text inside that `span` is tooltip text. This appears when you hover over the icon.

Reply:
Ah right, I should have looked more closely. The text appearing instead of the icon threw me.